packages/tables/resources/lang/nl/table.php

```diff
@@ -45,6 +45,14 @@
 
     'buttons' => [
 
+        'disable_reordering' => [
+            'label' => 'Ordenen afronden',
+        ],
+
+        'enable_reordering' => [
+            'label' => 'Opnieuw ordenen',
```

Comment:
I think this could just be:
```suggestion
            'label' => 'Records ordenen',
```